src/platform/item/wearable/wearable.ts

```diff
@@ -0,0 +1,99 @@
+import { generateValidator, JSONSchema } from '../../../validation'
+import { WearableCategory } from '../../../dapps/wearable-category'
+import { WearableRepresentation } from './representation'
+import {
+  BaseItem,
+  baseItemProperties,
+  requiredBaseItemProps
+} from '../base-item'
+import { StandardProps, standardProperties } from '../standard-props'
+import {
+  isThirdParty,
+  ThirdPartyProps,
+  thirdPartyProps
+} from '../third-party-props'
+
+/** @alpha */
+export type Wearable = BaseItem & {
+  data: {
+    replaces: WearableCategory[]
+    hides: WearableCategory[]
+    tags: string[]
+    representations: WearableRepresentation[]
+    category: WearableCategory
+  }
+} & (StandardProps | ThirdPartyProps)
+
+/** @alpha */
+export namespace Wearable {
+  export const schema: JSONSchema<Wearable> = {
+    type: 'object',
+    properties: {
+      ...baseItemProperties,
+      ...standardProperties,
+      ...thirdPartyProps,
+      data: {
+        type: 'object',
+        properties: {
+          replaces: {
+            type: 'array',
+            items: WearableCategory.schema
+          },
+          hides: {
+            type: 'array',
+            items: WearableCategory.schema
+          },
+          tags: {
+            type: 'array',
+            items: {
+              type: 'string',
+              minLength: 1
+            }
+          },
+          representations: {
+            type: 'array',
+            items: WearableRepresentation.schema,
+            minItems: 1
+          },
+          category: WearableCategory.schema
+        },
+        required: ['replaces', 'hides', 'tags', 'representations', 'category']
+      }
+    },
+    additionalProperties: true,
+    required: [...requiredBaseItemProps, 'data'],
+    oneOf: [
```

Comment:
Same here. Something along the lines of "either properties for a standard wearable or properties for third party wearable are required"